tests/zoomus/test_client.py

```diff
@@ -47,7 +47,17 @@ def test_cannot_init_with_non_int_timeout(self):
     def test_init_creates_all_components(self):
         client = ZoomClient("KEY", "SECRET")
         self.assertEqual(
-            set(["meeting", "past_meeting", "report", "user", "webinar", "recording"]),
+            set(
+                [
+                    "meeting",
```

Comment:
If you take a look at what is in develop now, these are now in alphabetical order. When you merge develop back in to this branch, can you please put `live_stream` at the appropriate place? Thanks.